src/main/java/com/joss/conductor/mobile/AccessibilityIdentifier.java

```diff
@@ -0,0 +1,13 @@
+package com.joss.conductor.mobile;
+
+public class AccessibilityIdentifier {
+    private String id;
+
+    public  AccessibilityIdentifier(String id) {
```

Comment:
Make this `private` and make a `static` method so that it follows the same convention as the `By` class, essentially you'd want this `ByAccessibilityIdentifier.id(someString);` to return you a new `ByAccessibilityIdentifier` object.

I added this comment again because it seems to have been buried by a "new" commit.